.gitignore

```diff
@@ -1,2 +1,3 @@
 target/
 Cargo.lock
+idea
```

Comment:
i think this will need to be removed from the final diff, but we can consider in another PR adding ignores for local configuration settings.